app-modules/knowledge-base/database/seeders/KnowledgeBaseArticleSeeder.php

```diff
@@ -46,5 +48,12 @@ public function run(): void
         KnowledgeBaseArticle::factory()
             ->count(25)
             ->create();
+
+        $settings = app(PortalSettings::class);
+        $settings->knowledge_management_portal_enabled = true;
+        $settings->knowledge_management_portal_primary_color = 'emerald';
+        $settings->knowledge_management_portal_rounding = Rounding::None->value;
+        $settings->knowledge_management_portal_service_management = true;
+        $settings->save();
```

Comment:
I get what you are doing here to try and save time. But let's not add these settings changes to the Seeder as enabling and configuring of the Portal does not really have to do with seeding KB articles.